api/routes/index.js

```diff
@@ -102,21 +109,21 @@ router.delete('/delete', async (req, res) => {
   }
 });
 
-router.post('/coop', async (req, res) => {
-  const { email, name, addr, pass } = req.body;
+router.post('/coop', isAuthenticated, async (req, res) => {
+  const { email, name, addr, hashed_pass } = req.body;
```

Comment:
nit: camelCase hashed_pass -> hashedPassword (make sure to change on frontend as well)